### PR TITLE
1.2 apis tools rename

### PR DESCRIPTION
**Pull request by christinaausley**
## What is the purpose of the change

APIs/Clients -> APIs/Tools rename for 1.2 docs.

## PR Checklist

- [x] My changes apply to an already released version, and I have added them to the relevant `/versioned_docs` directory, or they are not for an already released version.
- [x] My changes apply to future versions, and I have added them to the main `/docs` directory, or they are not for future versions.
- [x] My changes require an [Engineering review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned an engineering manager or tech lead as a reviewer, or my changes do not require an Engineering review.
- [x] My changes require a [technical writer review](https://github.com/camunda/camunda-platform-docs/blob/main/howtos/documentation-guidelines.md#review-process), and I've assigned @christinaausley as a reviewer, or my changes do not require a technical writer review.

### CODE REVIEW

**Comment by christinaausley** [comment]
@pepopowitz Mind taking a look at my redirect here? Once that's complete, I think this should build just fine.

**Comment by christinaausley** [comment]
@pepopowitz @akeller We have a clean build! This is ready for final review.

File: versioned_docs/version-1.2/apis-tools/overview.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ---
-# This page exists in apis-tools, instead of apis-clients, because the location needs to be consistent across all versions.
+# This page exists in apis-tools, instead of apis-tools, because the location needs to be consistent across all versions.
 id: working-with-apis-tools
 title: "Working with APIs & Clients"
```
Comment (akeller) [inline versioned_docs/version-1.2/apis-tools/overview.md:4]:
```suggestion
title: "Working with APIs & Tools"
```

**Review by pepopowitz — APPROVED** [review]
🚢

**Pull request merged**